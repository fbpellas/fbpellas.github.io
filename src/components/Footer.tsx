import * as React from 'react';
import { AUTHOR_FULLNAME, CREATION_YEAR_START, CREATION_YEAR_END } from '../constants';

export interface FooterProps { }

const Footer: React.FC<FooterProps> = (_props) => {
  return (
    <div className="footer">
      {AUTHOR_FULLNAME}, website created in {CREATION_YEAR_START}-{CREATION_YEAR_END} and hosted on{' '}
      <a
        className="clickable-page"
        target="_blank"
        rel="noreferrer noopener"
        href="https://github.com/fbpellas/fbpellas.github.io"
      >
        GitHub
      </a>
    </div>
  );
};

export { Footer };