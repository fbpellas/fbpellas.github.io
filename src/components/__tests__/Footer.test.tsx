import React from 'react';
import { render } from '@testing-library/react';
import { Footer } from '../Footer';

test('Footer', () => {
  const { container } = render(<Footer />);
  expect(container).toMatchSnapshot();
});
