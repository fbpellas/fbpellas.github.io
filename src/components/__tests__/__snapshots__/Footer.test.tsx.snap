// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer 1`] = `
<div>
  <div
    class="footer"
  >
    Faith Pellas
    , website created in 
    2020
    -
    2022
     and hosted on
     
    <a
      class="clickable-page"
      href="https://github.com/fbpellas/fbpellas.github.io"
      rel="noreferrer noopener"
      target="_blank"
    >
      GitHub
    </a>
  </div>
</div>
`;
